language/predefined/generator/throw.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<!-- EN-Revision: 8fee3ae9715ffa15922469eb7d98f4878917a6ee Maintainer: samesch Status: ready -->
+<!-- Reviewed: no -->
+<refentry xml:id="generator.throw" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>Generator::throw</refname>
+  <refpurpose>Wirft eine Exception in einen Generator</refpurpose>
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis role="Generator">
+   <modifier>public</modifier> <type>mixed</type><methodname>Generator::throw</methodname>
+   <methodparam><type>Throwable</type><parameter>exception</parameter></methodparam>
+  </methodsynopsis>
+  <para>
+   Wirft eine Exception in einen Generator und setzt die Ausführung des
+   Generators fort. Der Generator verhält sich dann so, als wäre der aktuelle
+   &yield;-Ausdruck durch eine <literal>throw $exception</literal>-Anweisung
+   ersetzt worden.
+  </para>
+  <para>
+   Wenn der Generator beim Aufruf dieser Methode bereits geschlossen ist, wird
+   die Exception stattdessen im Kontext des aufrufenden Codes geworfen.
```

Comment:
In my opinion, the word "Exception" does not need to be translated. But above (within `Fiber::isTerminated`) you have translated this word with "Ausnahme". We should keep this consistent here. 

I would replace "Ausnahme" with "Exception" above (within `Fiber::isTerminated`).

Reply:
Me too. Deepl always translates it.